docs/personalization/triggers.md

```diff
@@ -12,7 +12,8 @@ As a store manager, you can expect that more engaged customers translate to bigg
 There are several email triggers available in [[= product_name =]], including the following two:
 
 - Abandoned basket trigger: Personalization engine monitors the user's cart and pushes a message when cart status remains unchanged for a set time. 
-The message contains items that have been abandoned in the cart.
+The message contains items that have been abandoned in the cart. 
+To remove items that the user is no longer interested in from recommendations, the trigger logic monitors the `deletefrombasket` and `buy` events and filters out items that these events relate to.  
```

Comment:
I think it needs more explanation, as at first glance it seems hard to understand the logic behind this trigger.